### PR TITLE
Added a blurb to all relevant pages where USSC deprecation needs to be displayed.

### PR DESCRIPTION
**Pull request by GregHolmes**
## Description

I have updated all of the relevant pages where US Short Code is featured. These pages now have a blurb at the top announcing the deprecation of this service along with where the users can go from here. The blurb was decided by the team heading this deprecation work and is universal throughout multiple platforms that it will be displayed.

I need to get Lorenzo to approve of these changes, and a date to be given for it to be merged. But would like you to cast your eyes over this to see if you can see any problems or if there are any pages that you know of, that I may have missed out?

Thank you!

### CODE REVIEW

File: _api/sms/us-short-codes/alerts/sending.md [hunk 1]
```diff
@@ -6,6 +6,10 @@ api: Alerts API - Sending
 
 # Alerts - Sending API Reference
 
+> **Action Needed For Vonage Customers Using US Shared Short Codes**
+>
+> **Effective immediately, Vonage will no longer accept new programs for Shared Short Codes for A2P messaging.** T-Mobile and AT&T’s new code of conduct prohibits the use of shared originators, therefore, existing Shared Short Code traffic must be migrated by March 1, 2021. To help you with this transition, please use the Vonage [guide on alternatives](https://help.nexmo.com/hc/en-us/articles/360050905592).  Please contact us to migrate to a new solution.
```
Comment (marklewin) [inline _api/sms/us-short-codes/alerts/sending.md:11]:
```suggestion
> **Effective immediately, Vonage will no longer accept new programs for Shared Short Codes for A2P messaging.** T-Mobile and AT&T’s new code of conduct prohibits the use of shared originators, therefore, existing Shared Short Code traffic must be migrated by March 1, 2021. To help you with this transition, please use the Vonage [guide to alternatives](https://help.nexmo.com/hc/en-us/articles/360050905592).  Please contact us to migrate to a new solution.
```

File: _api/sms/us-short-codes/alerts/subscription.md [hunk 2]
```diff
@@ -6,6 +6,10 @@ api: Alerts API - Subscribing
 
 # Alerts - Subscribing API Reference
 
+> **Action Needed For Vonage Customers Using US Shared Short Codes**
+>
+> **Effective immediately, Vonage will no longer accept new programs for Shared Short Codes for A2P messaging.** T-Mobile and AT&T’s new code of conduct prohibits the use of shared originators, therefore, existing Shared Short Code traffic must be migrated by March 1, 2021. To help you with this transition, please use the Vonage [guide on alternatives](https://help.nexmo.com/hc/en-us/articles/360050905592).  Please contact us to migrate to a new solution.
```
Comment (marklewin) [inline _api/sms/us-short-codes/alerts/subscription.md:11]:
```suggestion
> **Effective immediately, Vonage will no longer accept new programs for Shared Short Codes for A2P messaging.** T-Mobile and AT&T’s new code of conduct prohibits the use of shared originators, therefore, existing Shared Short Code traffic must be migrated by March 1, 2021. To help you with this transition, please use the Vonage [guide to alternatives](https://help.nexmo.com/hc/en-us/articles/360050905592).  Please contact us to migrate to a new solution.
```

File: _documentation/en/concepts/guides/glossary.md [hunk 3]
```diff
@@ -6,6 +6,10 @@ navigation_weight: 6
 
 # Glossary
 
+## 10DLC
+
+10 digit long code (10DLC) is the new standard in Application 2 Person (A2P) text messaging. This standard allows businesses to have a higher volume throughput and improves security standards for messaging.
```
Comment (marklewin) [inline _documentation/en/concepts/guides/glossary.md:11]:
```suggestion
10 digit long code (10DLC) is the new standard in Application to Person (A2P) text messaging. This standard allows businesses to have a higher volume throughput and improves security standards for messaging.
```

File: _documentation/en/concepts/guides/glossary.md [hunk 4]
```diff
@@ -419,6 +423,10 @@ Further information can be found on the [Nexmo Tools page](https://developer.nex
 
 A Short Code that is instantly available to Customers. These US Short Codes are shared with other customers.
 
+## Shared Short Code Deprecation
+
+**Effective immediately, Vonage will no longer accept new programs for Shared Short Codes for A2P messaging.** T-Mobile and AT&T’s new code of conduct prohibits the use of shared originators, therefore, existing Shared Short Code traffic must be migrated by March 1, 2021. To help you with this transition, please use the Vonage [guide on alternatives](https://help.nexmo.com/hc/en-us/articles/360050905592).  Please contact us to migrate to a new solution.
```
Comment (marklewin) [inline _documentation/en/concepts/guides/glossary.md:428]:
```suggestion
**Effective immediately, Vonage will no longer accept new programs for Shared Short Codes for A2P messaging.** T-Mobile and AT&T’s new code of conduct prohibits the use of shared originators, therefore, existing Shared Short Code traffic must be migrated by March 1, 2021. To help you with this transition, please use the Vonage [guide to alternatives](https://help.nexmo.com/hc/en-us/articles/360050905592).  Please contact us to migrate to a new solution.
```

File: _documentation/en/dispatch/code-snippets/send-an-mms-with-failover.md [hunk 5]
```diff
@@ -4,6 +4,10 @@ title: Send an MMS with failover
 
 # Send an MMS with failover
 
+> **Action Needed For Vonage Customers Using US Shared Short Codes**
+>
+> **Effective immediately, Vonage will no longer accept new programs for Shared Short Codes for A2P messaging.** T-Mobile and AT&T’s new code of conduct prohibits the use of shared originators, therefore, existing Shared Short Code traffic must be migrated by March 1, 2021. To help you with this transition, please use the Vonage [guide on alternatives](https://help.nexmo.com/hc/en-us/articles/360050905592).  Please contact us to migrate to a new solution.
```
Comment (marklewin) [inline _documentation/en/dispatch/code-snippets/send-an-mms-with-failover.md:9]:
```suggestion
> **Effective immediately, Vonage will no longer accept new programs for Shared Short Codes for A2P messaging.** T-Mobile and AT&T’s new code of conduct prohibits the use of shared originators, therefore, existing Shared Short Code traffic must be migrated by March 1, 2021. To help you with this transition, please use the Vonage [guide to alternatives](https://help.nexmo.com/hc/en-us/articles/360050905592).  Please contact us to migrate to a new solution.
```

File: _documentation/en/messages/code-snippets/mms/send-mms.md [hunk 6]
```diff
@@ -5,6 +5,10 @@ meta_title: Send an MMS using the Messages API
 
 # Send an MMS
 
+> **Action Needed For Vonage Customers Using US Shared Short Codes**
+>
+> **Effective immediately, Vonage will no longer accept new programs for Shared Short Codes for A2P messaging.** T-Mobile and AT&T’s new code of conduct prohibits the use of shared originators, therefore, existing Shared Short Code traffic must be migrated by March 1, 2021. To help you with this transition, please use the Vonage [guide on alternatives](https://help.nexmo.com/hc/en-us/articles/360050905592).  Please contact us to migrate to a new solution.
```
Comment (marklewin) [inline _documentation/en/messages/code-snippets/mms/send-mms.md:10]:
```suggestion
> **Effective immediately, Vonage will no longer accept new programs for Shared Short Codes for A2P messaging.** T-Mobile and AT&T’s new code of conduct prohibits the use of shared originators, therefore, existing Shared Short Code traffic must be migrated by March 1, 2021. To help you with this transition, please use the Vonage [guide to alternatives](https://help.nexmo.com/hc/en-us/articles/360050905592).  Please contact us to migrate to a new solution.
```

File: _documentation/en/messaging/us-short-codes/guides/2fa.md [hunk 7]
```diff
@@ -4,6 +4,10 @@ title: 2FA
 
 # Two-factor Authentication API
 
+> **Action Needed For Vonage Customers Using US Shared Short Codes**
+>
+> **Effective immediately, Vonage will no longer accept new programs for Shared Short Codes for A2P messaging.** T-Mobile and AT&T’s new code of conduct prohibits the use of shared originators, therefore, existing Shared Short Code traffic must be migrated by March 1, 2021. To help you with this transition, please use the Vonage [guide on alternatives](https://help.nexmo.com/hc/en-us/articles/360050905592).  Please contact us to migrate to a new solution.
```
Comment (marklewin) [inline _documentation/en/messaging/us-short-codes/guides/2fa.md:9]:
```suggestion
> **Effective immediately, Vonage will no longer accept new programs for Shared Short Codes for A2P messaging.** T-Mobile and AT&T’s new code of conduct prohibits the use of shared originators, therefore, existing Shared Short Code traffic must be migrated by March 1, 2021. To help you with this transition, please use the Vonage [guide to alternatives](https://help.nexmo.com/hc/en-us/articles/360050905592).  Please contact us to migrate to a new solution.
```

File: _documentation/en/messaging/us-short-codes/guides/alerts.md [hunk 8]
```diff
@@ -4,6 +4,10 @@ title: Alerts
 
 # Alerts
 
+> **Action Needed For Vonage Customers Using US Shared Short Codes**
+>
+> **Effective immediately, Vonage will no longer accept new programs for Shared Short Codes for A2P messaging.** T-Mobile and AT&T’s new code of conduct prohibits the use of shared originators, therefore, existing Shared Short Code traffic must be migrated by March 1, 2021. To help you with this transition, please use the Vonage [guide on alternatives](https://help.nexmo.com/hc/en-us/articles/360050905592).  Please contact us to migrate to a new solution.
```
Comment (marklewin) [inline _documentation/en/messaging/us-short-codes/guides/alerts.md:9]:
```suggestion
> **Effective immediately, Vonage will no longer accept new programs for Shared Short Codes for A2P messaging.** T-Mobile and AT&T’s new code of conduct prohibits the use of shared originators, therefore, existing Shared Short Code traffic must be migrated by March 1, 2021. To help you with this transition, please use the Vonage [guide to alternatives](https://help.nexmo.com/hc/en-us/articles/360050905592).  Please contact us to migrate to a new solution.
```

File: _documentation/en/messaging/us-short-codes/guides/campaign-subscription-management.md [hunk 9]
```diff
@@ -5,6 +5,10 @@ description: Create an opt-in and opt-out process for recipients of your campaig
 
 # Campaign Subscription Management
 
+> **Action Needed For Vonage Customers Using US Shared Short Codes**
+>
+>**Effective immediately, Vonage will no longer accept new programs for Shared Short Codes for A2P messaging.** T-Mobile and AT&T’s new code of conduct prohibits the use of shared originators, therefore, existing Shared Short Code traffic must be migrated by March 1, 2021. To help you with this transition, please use the Vonage [guide on alternatives](https://help.nexmo.com/hc/en-us/articles/360050905592).  Please contact us to migrate to a new solution.
```
Comment (marklewin) [inline _documentation/en/messaging/us-short-codes/guides/campaign-subscription-management.md:10]:
```suggestion
>**Effective immediately, Vonage will no longer accept new programs for Shared Short Codes for A2P messaging.** T-Mobile and AT&T’s new code of conduct prohibits the use of shared originators, therefore, existing Shared Short Code traffic must be migrated by March 1, 2021. To help you with this transition, please use the Vonage [guide to alternatives](https://help.nexmo.com/hc/en-us/articles/360050905592).  Please contact us to migrate to a new solution.
```

File: _documentation/en/messaging/us-short-codes/overview.md [hunk 10]
```diff
@@ -6,6 +6,10 @@ description: Vonage provides an API for sending SMS messages from a shared short
 
 # US Short Codes Overview
 
+> **Action Needed For Vonage Customers Using US Shared Short Codes**
+>
+>**Effective immediately, Vonage will no longer accept new programs for Shared Short Codes for A2P messaging.** T-Mobile and AT&T’s new code of conduct prohibits the use of shared originators, therefore, existing Shared Short Code traffic must be migrated by March 1, 2021. To help you with this transition, please use the Vonage [guide on alternatives](https://help.nexmo.com/hc/en-us/articles/360050905592).  Please contact us to migrate to a new solution.
```
Comment (marklewin) [inline _documentation/en/messaging/us-short-codes/overview.md:11]:
```suggestion
>**Effective immediately, Vonage will no longer accept new programs for Shared Short Codes for A2P messaging.** T-Mobile and AT&T’s new code of conduct prohibits the use of shared originators, therefore, existing Shared Short Code traffic must be migrated by March 1, 2021. To help you with this transition, please use the Vonage [guide to alternatives](https://help.nexmo.com/hc/en-us/articles/360050905592).  Please contact us to migrate to a new solution.
```

File: _documentation/en/numbers/guides/enable-2fa.md [hunk 11]
```diff
@@ -6,6 +6,10 @@ navigation_weight: 2
 
 # Enable Two-Factor Authentication
 
+> **Action Needed For Vonage Customers Using US Shared Short Codes**
+>
+> **Effective immediately, Vonage will no longer accept new programs for Shared Short Codes for A2P messaging.** T-Mobile and AT&T’s new code of conduct prohibits the use of shared originators, therefore, existing Shared Short Code traffic must be migrated by March 1, 2021. To help you with this transition, please use the Vonage [guide on alternatives](https://help.nexmo.com/hc/en-us/articles/360050905592).  Please contact us to migrate to a new solution.
```
Comment (marklewin) [inline _documentation/en/numbers/guides/enable-2fa.md:11]:
```suggestion
> **Effective immediately, Vonage will no longer accept new programs for Shared Short Codes for A2P messaging.** T-Mobile and AT&T’s new code of conduct prohibits the use of shared originators, therefore, existing Shared Short Code traffic must be migrated by March 1, 2021. To help you with this transition, please use the Vonage [guide to alternatives](https://help.nexmo.com/hc/en-us/articles/360050905592).  Please contact us to migrate to a new solution.
```

File: _documentation/en/numbers/guides/event-alerts.md [hunk 12]
```diff
@@ -8,7 +8,9 @@ navigation_weight: 3
 
 You can use a short code to send event alerts or reminders to your users - for example, to remind them of an appointment. The [Event-based alerts API](/api/sms/us-short-codes/alerts/sending) provides this capability. 
 
-> **Note** Our carrier partner (AT&T) is discontinuing its shared short code service for alerts. Nexmo is therefore no longer able to offer this service. [Dedicated short codes](https://www.nexmo.com/platform/short-codes) are still available.
+> **Action Needed For Vonage Customers Using US Shared Short Codes**
+>
+> **Effective immediately, Vonage will no longer accept new programs for Shared Short Codes for A2P messaging.** T-Mobile and AT&T’s new code of conduct prohibits the use of shared originators, therefore, existing Shared Short Code traffic must be migrated by March 1, 2021. To help you with this transition, please use the Vonage [guide on alternatives](https://help.nexmo.com/hc/en-us/articles/360050905592).  Please contact us to migrate to a new solution.
```
Comment (marklewin) [inline _documentation/en/numbers/guides/event-alerts.md:13]:
```suggestion
> **Effective immediately, Vonage will no longer accept new programs for Shared Short Codes for A2P messaging.** T-Mobile and AT&T’s new code of conduct prohibits the use of shared originators, therefore, existing Shared Short Code traffic must be migrated by March 1, 2021. To help you with this transition, please use the Vonage [guide to alternatives](https://help.nexmo.com/hc/en-us/articles/360050905592).  Please contact us to migrate to a new solution.
```

**Review by marklewin — APPROVED** [review]
Very minor grammatical changes x 2, but otherwise LGTM!

**Review by marklewin — CHANGES_REQUESTED** [review]
Sorry, meant to mark as 'request changes'

**Review by marklewin — APPROVED** [review]
LGTM!

**Comment by GregHolmes** [comment]
Hey @marklewin , is there any chance this could be deployed whenever you have time and an active slot to deploy please?

**Pull request merged**